app/src/androidTest/java/com/example/campusguide/CurrentLocationAcceptanceTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.example.campusguide
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.matcher.ViewMatchers.assertThat
+import androidx.test.filters.SdkSuppress
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.uiautomator.*
+import junit.framework.Assert.assertEquals
+import org.hamcrest.CoreMatchers.notNullValue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+private const val LAUNCH_TIMEOUT = 5000L
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@SdkSuppress(minSdkVersion = 18)
```

Comment:
What's this for?

Reply:
ensures that the tests will only run on Android 4.3 or higher (which is required for UI automator)